packages/prop-house-webapp/package.json

```diff
@@ -62,14 +62,15 @@
     "sanitize-html": "^2.7.0",
     "slugify": "^1.6.5",
     "swiper": "^9.1.0",
-    "typescript": "^4.8.4",
-    "wagmi": "^0.9.2",
+    "typescript": "5.1",
+    "viem": "^1.5.3",
+    "wagmi": "^1.2",
     "web-vitals": "^1.0.1",
     "web3modal": "^1.9.5",
     "yarn": "^1.22.18"
   },
   "scripts": {
-    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
@cryptoseneca is this vital? i couldn't get the app running until i removed

Reply:
relevant info:

- https://github.com/Prop-House/prop-house-monorepo/commit/b38a6bd128dfc0e41290a7ba1fa72f4fab992bf8
- https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported
- https://github.com/webpack/webpack/issues/14532